### PR TITLE
Replace Travis CI with GitHub Actions

### PR DESCRIPTION
**Pull request by andyw8**
https://www.travis-ci.org/ "will be shutting down in several weeks".

Although this could be  moved to travis-ci.com, GitHub Actions is a better option IMO.

TODO:
- [x] wait for #85 so the tests are passing
- [x] update the badge in the README
- [ ] An admin may need to enable GitHub Actions in this repo.

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['1.9', '2.7']
```
Comment (andyw8) [inline .github/workflows/ruby.yml:15]: Ruby 3.0 doesn't work yet, see #87.

**Comment by cantino** [comment]
Let me know when this is ready for review.

**Comment by andyw8** [comment]
@cantino Ready

**Comment by andyw8** [comment]
Thanks for fixing up the badge and versions 👍

**Comment by cantino** [comment]
Thanks for the improvements!

**Pull request merged**